src/sources.js

```diff
@@ -333,7 +338,8 @@ class S3Source extends Source {
         title: `"${this.name}" Nextstrain group`,
         byline: `The available datasets and narratives in this group are listed below.`,
         showDatasets: true,
-        showNarratives: true
+        showNarratives: true,
+        avatar: await this.availableLogo()
```

Comment:
It feels strange for this to be in the "Appropriate fallback if no customised data is available" catch-block rather than the "attempt to fetch customisable information from S3 bucket" block.

Reply:
Yup, updated in the new commit